maze_transformer/evaluation/plot_maze.py

```diff
@@ -6,20 +6,113 @@
 
 from maze_transformer.generation.latticemaze import LatticeMaze
 
+CoordArray = NDArray["coords", np.int8]
+
+# Set size of unit.
+# Each "unit" consists of a node and the right and lower adjacent wall/connection. Its area is ul * ul.
+# Wall thickness is fixed to 1, so this parameter sets the ratio of node length to wall thickness
+ul = 14
+
+
+# Maze conversion functions (No plt involved)
```

Comment:
I guess you are doing this outside because as_img and points_transform_to_img both depend on it? 

In that case I think we should introduce a class.

We could have a nice [fluent interface](https://martinfowler.com/bliki/FluentInterface.html) like:
```
MazePlot(maze).add_path(path).add_path(path).show()
```
or
```
MazePlot(maze).add_multi_paths(path).show()
```


Let me know if you want to pair on this. Also let me know if i am over-engineering it! 😅

Reply:
good suggestion, I implemented the fluent interface!